### PR TITLE
feat: add new project credhub

### PR DESCRIPTION
**Pull request by cre8**
Add the project proposal to add credhub to the Open Wallet Foundation

### CODE REVIEW

File: projects/dori.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Project Name
+DORI: Decentralized Online Resource for Identity
+
+# Preferred Maturity Level
+Labs
+
+# Project Description
+When talking about wallets for natural persons, a lot of people think that a smartphone based wallet is the only way to go. The user should feel the security and privacy by being independent from an external provider. But we are forgetting that online based services have a great user experience in daily live:
+As a user I can use multiple clients where my data is synced, we do not have to care about backups or data recovery.
+As a developer I can implement the business logic into the cloud and only need the rendering on the client side.
```
Comment (tkuhrt) [inline projects/dori.md:10]:
```suggestion
When talking about wallets for natural persons, a lot of people think that a smartphone based wallet is the only way to go. The user should feel the security and privacy by being independent from an external provider. But we are forgetting that online based services have a great user experience in daily life:
- As a user I can use multiple clients where my data is synced, we do not have to care about backups or data recovery.
- As a developer I can implement the business logic into the cloud and only need the rendering on the client side.
```

File: projects/credhub.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+[OpenWallet Foundation code of conduct](https://tac.openwallet.foundation/governance/code-of-conduct/)
+
+# TAC Sponsor
+None
+
+# Project License
+Apache 2.0 for all projects inside the Monorepo.
+
+# Source Control
+[Github Repo](https://github.com/cre8/wallet)
```
Comment (skounis) [inline projects/credhub.md:58]: Is there a plan to transfer the repository to the OWF Organization?

File: projects/credhub.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+It can also be used as a playground to test out new approaches: how can processes be automated for the best UX without a security tradeoff? Implementing alterative approaches like another signature algorithm and compare them with the current implementation.
+
+# Code of Conduct
+[OpenWallet Foundation code of conduct](https://tac.openwallet.foundation/governance/code-of-conduct/)
+
+# TAC Sponsor
+None
+
+# Project License
+Apache 2.0 for all projects inside the Monorepo.
```
Comment (skounis) [inline projects/credhub.md:55]: Do all the dependencies come with a compatible non copyleft license? 

https://github.com/cre8/wallet/blob/main/package.json

File: projects/credhub.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+Labs
+
+# Project Description
+When talking about wallets for natural persons, a lot of people think that a smartphone based wallet is the only way to go. The user should feel the security and privacy by being independent from an external provider. But we are forgetting that online based services have a great user experience in daily life:
+- As a user I can use multiple clients where my data is synced, we do not have to care about backups or data recovery.
+- As a developer I can implement the business logic into the cloud and only need the rendering on the client side.
+
+Yes, when I am offline, I do not have access to my data anymore. But beside that the user experience is great!
+
+The crehub project is a typescript based monorepo, using [nx](https://nx.dev) to manage multiple components. Compared to other approaches, it also comes with an issuer and verifier service to issue and verify credentials.
```
Comment (skounis) [inline projects/credhub.md:14]: Should we for consistency capitalize the names of the projects, e.g. "Crehub"?

File: projects/credhub.md (hunk 2 above)
Comment (skounis) [reply projects/credhub.md:58]: Despite the move or not should we change the machine name of the repo from `cre8` to `credhub` for consistency?

Comment (cre8) [reply projects/credhub.md:58]: The repository should be moved to the OWF repository and be named credhub. For now there is not planned to have multiple repositories that require to host the project via a dedicated organisation.

File: projects/credhub.md (hunk 3 above)
Comment (cre8) [reply projects/credhub.md:55]: @skounis running [license checker](https://www.npmjs.com/package/license-checker) on the repository and removing `private: true` from the package.json, it returns these license information for all dependencies (dev and prod):
```
├─ MIT: 97
├─ Apache-2.0: 25
├─ BSD-2-Clause: 3
├─ BSD-3-Clause: 2
└─ 0BSD: 1
```
So all licences should be aligned with the Apache 2.0 license for this project.

I will open an issue to include this check into the CICD so a new dependencies will not bring any trouble

**Comment by tkuhrt** [comment]
APPROVED at the May 29, 2024 meeting

**Pull request merged**